package.json

```diff
@@ -9,7 +9,9 @@
   },
   "scripts": {
     "build": "webpack -d --watch",
-    "seed": "mongoimport --db carousels --collection Carousel --file ./database/carousel-data.json --jsonArray",
+    "init:db": "psql -d nearby_restaurants -f database/schema.sql",
+    "seed:csv": "node --max-old-space-size=8192 database/datamocker.js",
```

Comment:
Does this mean your script works only if you have >= 8GB of RAM? I'm a bit worried that you won't be able to rely on this when deploying. Anyway, this is something to be aware of, even if you don't have time to refactor right now.

Reply:
Oh my. This is an interesting point. I'll have to test it out at a lower limit like 2GB and see how it goes.